lib/code_corps/policy/policy.ex

```diff
@@ -35,8 +38,9 @@ defmodule CodeCorps.Policy do
     # will never do anything
     #
     # The only solution is to have a catch_all match for the resource being nil, which returns true
+    
+    # NOTE: other tests are using the User policy for the time being.
     def can?(%User{}, _action, nil), do: true
-
     def can?(%User{} = current_user, :update, %User{} = user), do: Policy.User.update?(user, current_user)
```

Comment:
NOTE: had to leave these in here b/c other tests not in the `user_controller_test` were depending on this

Reply:
I understand why line 44 needs to stay, but are you sure about 45? Which tests fail if you remove it? Assuming you add the "fixes" i described below.